cluster-autoscaler/core/utils_test.go

```diff
@@ -109,10 +109,11 @@ func testSanitizeLabels(t *testing.T) {
 	node := BuildTestNode("ng1-1", 1000, 1000)
 	node.Labels[metav1.LabelHostname] = "abc"
 	node.Labels["x"] = "y"
-	node, err := sanitizeTemplateNodeLables(node)
+	node, err := sanitizeTemplateNode(node, "bzium")
 	assert.NoError(t, err)
 	assert.NotEqual(t, node.Labels[metav1.LabelHostname], "abc")
 	assert.Equal(t, node.Labels["x"], "y")
+	assert.NotEqual(t, node.Name, "ng1-1")
```

Comment:
How is that test passing if we're don't seem to be changing node.Name? 

Anyway I would like to add here:
```
assert.Equal(t, node.Name, node.Labels[metav1.LabelHostname])
```
Or is that not an expected condition for some reason?